db/models.js

```diff
@@ -1,35 +1,36 @@
 // Static methods for database interactions
 class Models {
   static getCampaignById(db, id) {
-    return db.query('SELECT * FROM campaigns WHERE id = ?', [id])
+    return db.query('SELECT * FROM campaigns WHERE id = $1;', [id])
+      .then((result) => result.rows[0])
       .catch((err) => { console.error(err); });
   }
 
-  static getCampaignsByAuthor(db, author) {
-    return db.query('SELECT * FROM campaigns WHERE author = ?', [author])
+  static getCampaignsByUser(db, username) {
+    return db.query('SELECT * FROM campaigns WHERE _user = $1;', [username])
+      .then((result) => result.rows)
       .catch((err) => { console.error(err); });
```

Comment:
`  .catch((err) => console.error(err));`

Definitely a nitpicky suggestion, figured I'd leave it anyway.